### PR TITLE
Added Docker.md

### PR DESCRIPTION
**Pull request by jorgebsa**
Added a basic guide on how to run PokéAlarm with docker. Let me know what you think and how we can improve this! 

I have been running PokéAlarm on docker for over a week now. However, given the great amount of copy-paste and replacing variables from my commands involved in creating this, it is susceptible to errors lol, overall it should be fine though ;)

### CODE REVIEW

File: Docker.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  * [PokémonGo-Map running without docker](#pokémongo-map-running-without-docker)
+* [Updating alarms](#updating-alarms)
+* [Running multiple alarm containers](#running-multiple-alarm-containers)
+
+## Prerequisites
+
+This guide assumes: 
+
+1) You have [Docker](https://docs.docker.com/) installed and running on your machine
+2) You have [Git](https://git-scm.com/downloads) installed and running on your machine
```
Comment (kvangent) [inline Docker.md]: Needs a linebreak at the end

File: Docker.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+2) You have read and understood the [Alarm Configuration](https://github.com/kvangent/PokeAlarm/wiki/Alarm-Configuration) Wiki
```
Comment (kvangent) [inline Docker.md]: Should be `3)`

File: Docker.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+The first thing that you need to do is to clone the PokéAlarm repository in the directory you'd like:
+
+```
+git clone https://github.com/kvangent/PokeAlarm.git
+```
+
+### Configuring the alarm
+
+Now, enter the repository's directory and copy the file called `alarms.json.default` to a file named `alarms.json`. Once that is done, open `alarms.json` and edit it to your liking. Refer to the alarm configuration documentation linked above as needed.
```
Comment (kvangent) [inline Docker.md]: Can we add a short line about editing `config/config.ini` as well? Make sure they don't need to add command line parameters then.

**Comment by jorgebsa** [comment]
@kvangent implemented the requested changes, please take a look 👍

**Pull request merged**